drivers/gpio/gpio_rpi_pico.c

```diff
@@ -50,6 +58,11 @@ static int gpio_rpi_configure(const struct device *dev,
 	/* Avoid gpio_init, since that also clears previously set direction/high/low */
 	gpio_set_function(pin, GPIO_FUNC_SIO);
 
+	if (flags & GPIO_INPUT) {
+		gpio_set_dir(pin, GPIO_IN);
+	} else {
+		gpio_set_input_enabled(pin, false);
+	}
```

Comment:
Insert blank.

Reply:
It's a matter of habit. When you look at other code, it's obvious that line breaks are often inserted after `}` and you probably know that too.
The role of reviews is to check for areas that aren't flagged by automated checks.

If I try to give a clear reason, it forms a single semantic unit as a block and has a relationship of parallelism, contraposition, etc. with the next phrase.